source/puppet/5.3/hiera_automatic.md

```diff
@@ -29,57 +29,60 @@ For example, you can set servers for the `NTP` class like this:
 ntp::servers:
   - time.example.com
   - 0.pool.ntp.org
-  ```
```

Comment:
Opening and closing code fences should be indented consistently. The opening fence was not indented, but the closing fence was. This caused the code block to render incorrectly.